NebulaHost/NebulaHost.csproj

```diff
@@ -37,6 +37,9 @@
     <Reference Include="Assembly-CSharp">
       <HintPath>..\Libs\Assembly-CSharp.dll</HintPath>
     </Reference>
+    <Reference Include="LZ4, Version=1.0.15.93, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
Please use PackageReference.

Reply:
I just added the package normally via VS, no idea why it did it this way. Will try to move it over in the new patchset.

Reply:
Also, I think that the LZ4 lib should be a separate mod, simply because it might break in case multiple mods ship lz4.

Reply:
Will sort.

I think for now it's okay with the LZ4 lib, I imagine this mod is going to be setting on a lot of toes in regards to other mods with that amount of method patching that we do - everything is contained in the "Nebula" folder already within the plugins folder. If it becomes an issue, then we can fix it.